charts/prometheus-adapter/templates/deployment.yaml

```diff
@@ -55,6 +55,9 @@ spec:
         - --logtostderr=true
         - --prometheus-url={{ tpl .Values.prometheus.url . }}{{ if .Values.prometheus.port }}:{{ .Values.prometheus.port }}{{end}}{{ .Values.prometheus.path }}
         - --metrics-relist-interval={{ .Values.metricsRelistInterval }}
+        {{- if .Values.metricsMaxAge }}
+        - --metrics-max-age={{ .Values.metricsMaxAge }}
```

Comment:
Why do we need an explicit value for this when we have `extraArguments` that you can use to set `--metrics-max-age=`? I'm not in favor of more properties when we have an existing mechanism to support your use case.

Reply:
IMHO I think this metric is important because it comes hand in hand with External metrics. If the Helm chart should work out of the box with all the API's it is important that this property will be explicitly listed. 
It took me a while to understand why the Stack driver metrics don't appear even if all is working properly

Reply:
I'm not arguing whether the flag is useful or not, I'm arguing that you can already configure the current chart with that flag. Since you're adding a disabled by default flag there is little difference in people discovering the flag in the `values.yaml` or finding in the official Prometheus Adapter [documentation](https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/README.md#configuration). It does not add any value and I'm a proponent of keeping things simple.